### PR TITLE
Feat/algebraic polynomial generation modsub component

### PR DESCRIPTION
**Pull request by shrawant**

### CODE REVIEW

File: claasp/components/modsub_component.py [hunk 1]
```diff
@@ -39,6 +38,72 @@ def __init__(self, current_round_number, current_round_number_of_components,
...
+        EXAMPLES::
+
+            sage: from claasp.cipher_modules.models.algebraic.algebraic_model import AlgebraicModel
+            sage: from claasp.cipher import Cipher
+            sage: cipher = Cipher("cipher_name", "permutation", ["input"], [8], 8)
+            sage: cipher.add_round()
+            sage: modsub_0_0 = cipher.add_MODSUB_component(["input","input"], [[0,1,2,3],[4,5,6,7]], 4)
+            sage: modsub_component = cipher.get_component_from_id('modsub_0_0')
+            sage: algebraic = AlgebraicModel(cipher)
+            sage: modsub_component.algebraic_polynomials(algebraic)
```
Comment (peacker) [inline claasp/components/modsub_component.py:58]: report the output of the example

**Review by peacker — CHANGES_REQUESTED** [review]

File: claasp/components/modsub_component.py (hunk 1 above)
Comment (shrawant) [reply claasp/components/modsub_component.py:58]: Done

Comment (peacker) [reply claasp/components/modsub_component.py:58]: are you sure, I still do not see the output of line 58             
```sage: modsub_component.algebraic_polynomials(algebraic)```

Comment (shrawant) [reply claasp/components/modsub_component.py:58]: it is there

**Review by peacker — APPROVED** [review]

**Pull request merged**